crates/mun_compiler/src/driver.rs

```diff
@@ -50,10 +51,7 @@ impl Driver {
     }
 
     /// Constructs a driver with a configuration and a single file.
-    pub fn with_file(
-        config: Config,
-        path: PathOrInline,
-    ) -> Result<(Driver, FileId), failure::Error> {
+    pub fn with_file(config: Config, path: PathOrInline) -> Result<(Driver, FileId)> {
```

Comment:
```suggestion
    pub fn with_file(config: Config, path: PathOrInline) -> Result<(Driver, FileId), anyhow::Error> {
```

Reply:
Beware of potential `rustfmt` errors, due to too many characters on a single line